make: spurious Timeout exceeded

As of 53c7323 running `make` on Mac timeouts

```
Step 9/9 : COPY . .
 ---> Using cache
 ---> 330478b7b567
Successfully built 330478b7b567
Successfully tagged containous/maesh:check
docker run --rm \
      -v /Users/ysz/maesh:/go/src/github.com/containous/maesh \
      -w /go/src/github.com/containous/maesh \
      -e GO111MODULE \
      "containous/maesh:check" golangci-lint run --config .golangci.toml
level=error msg="Running error: context loading failed: no go files to analyze"
level=error msg="Timeout exceeded: try increase it by passing --timeout option"
make: *** [check] Error 4
```

Running in container gives

```
$ docker run -it -v /Users/ysz/maesh:/go/src/github.com/containous/maes -w /go/src/github.com/containous/maesh -e GO111MODULE "containous/maesh:check" bash
bash-5.0# golangci-lint -v run --config .golangci.toml
INFO [config_reader] Used config file .golangci.toml 
INFO [lintersdb] Active 29 linters: [bodyclose deadcode depguard dogsled errcheck gocognit goconst gocritic gocyclo godox gofmt goimports golint gosimple govet ineffassign interfacer misspell nakedret staticcheck structcheck stylecheck typecheck unconvert unparam unused varcheck whitespace wsl] 
INFO [loader] Go packages loading at mode 575 (files|name|compiled_files|exports_file|imports|types_sizes|deps) took 52.56940413s 
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 15.260549ms 
INFO Memory: 601 samples, avg is 86.1MB, max is 271.0MB 
INFO Execution took 1m0.199752748s                
INFO [runner] Issues before processing: 36, after processing: 0 
INFO [runner] Processors filtering stat (out/in): cgo: 36/36, skip_dirs: 36/36, identifier_marker: 36/36, exclude: 0/36, path_prettifier: 36/36, filename_unadjuster: 36/36, skip_files: 36/36, autogenerated_exclude: 36/36 
INFO [runner] processing took 5.368023ms with stages: identifier_marker: 3.947476ms, path_prettifier: 591.214µs, autogenerated_exclude: 370.065µs, skip_dirs: 195.463µs, exclude: 133.213µs, cgo: 83.036µs, filename_unadjuster: 37.137µs, max_from_linter: 5.017µs, max_same_issues: 2.187µs, nolint: 721ns, skip_files: 479ns, diff: 430ns, path_shortener: 420ns, exclude-rules: 342ns, source_code: 339ns, uniq_by_line: 275ns, max_per_file_from_linter: 209ns 
INFO [runner] linters took 36.07994956s with stages: goanalysis_metalinter: 34.045935977s, unused: 2.027864313s 
INFO File cache stats: 30 entries of total size 187.4KiB 
ERRO Timeout exceeded: try increase it by passing --timeout option 
bash-5.0# 
```

--- a/.golangci.toml
+++ b/.golangci.toml
@@ -1,5 +1,5 @@
 [run]
-  deadline = "5m"
+  timeout = "5m"
 
 [linters-settings]
 

